main.py

```diff
@@ -15,7 +15,7 @@
 import requests
 
 waka_key = os.getenv("INPUT_WAKATIME_API_KEY")
-
+stats_range = os.getenv("INPUT_STATS_RANGE", "last_7_days")
 
 def this_week(dates: list) -> str:

```

Comment:
```suggestion
def this_range(dates: list) -> str:

```

Reply:
fixed 👍 
87fe8c9b2c90586795ad60af5317f71a6a5a5a73

Reply:
I found similar places other than the one you pointed out.
So I changed `week` to `range` in the overall name.